x-pack/plugins/security/public/plugin.tsx

```diff
@@ -179,9 +187,11 @@ export class SecurityPlugin
 
     this.analyticsService.start({ http: core.http });
 
+    const navControlService = this.navControlService.start({ core, authc: this.authc });
+
     return {
       uiApi: getUiApi({ core }),
-      navControlService: this.navControlService.start({ core, authc: this.authc }),
+      navControlService,
```

Comment:
Nit: Is this change the accidental result of a debug-session or? Could be reversed unless you think it's better for readability

Reply:
Good catch! It was a leftover of a previous change that I moved to a separate plugin. Refer to https://github.com/elastic/kibana/pull/141103#discussion_r983293571 for context. Reverted that change :)